python/pipeline/pupil.py

```diff
@@ -1,3 +1,18 @@
+"""
+Date: 10/29/2019
+This module is for pupil related tables and computation.
+Current version is at 1.0.0 
+
+What's changed in this version:
```

Comment:
maybe you can add the date of the current version to make it easier to search later on Github

Reply:
updated